examples/with-react-router-4/src/server.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import express from 'express';
+import { renderToString } from 'react-dom/server';
+import { StaticRouter } from 'react-router-dom';
+import RootApp from './App';
+
+const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);
+
+const server = express();
+server
+  .disable('x-powered-by')
+  .use(express.static(process.env.RAZZLE_PUBLIC_DIR));
+
+server.use((req, res) => {
+  const routerContext = {};
+  const ServerApp = (
+    <StaticRouter location={req.url} context={routerContext}>
+      <RootApp />
+    </StaticRouter>
+  );
+
+  const markup = renderToString(ServerApp);
+
+  if (routerContext.url) {
+    res.status(302).setHeader('Location', routerContext.url);
```

Comment:
I think you should use `routerContext.statusCode` instead of `302` here
```suggestion
    res.redirect(routerContext.statusCode, routerContext.url);
```